### DISCUSSION TITLE
How to apply licence

### DISCUSSION THREAD

**Discussion by PavloParafiloCricut** [Megabit/Blazorise#4573] [Q&A] [answered]
Hi
Noticed message in console "Thank you for using the free version of the Blazorise component library! We're happy to offer it to you for personal use. If you'd like to remove this message, consider purchasing a commercial license"
Assuming, Blazorise is already purchased, how to get rid of that message? I mean, is it some licence key or is it some private nuget?
Thanks

**Comment by stsrki** [comment, marked as answer]
Hello, we have the license setup guide on our main web documentation. It can be found at https://blazorise.com/docs/usage/licensing/register-product-token

**Reply by PavloParafiloCricut** [reply]
Thanks. Sorry, missed that page
